docs/designers-developers/developers/themes/theme-json.md

```diff
@@ -2,35 +2,113 @@
 
 ### Settings can be controlled per block
 
-The Block Editor already allows the control of specific settings such as alignment, drop cap, whether it's present in the inserter, etc at the block level. The goal is to surface these for themes to control at a block level.
+The Block Editor already allows the control of specific settings such as alignment, drop cap, presets available, etc. All of these work at the block level. By using the `experimental-theme.json` we aim to allow themes to control these at a block level.
 
-### CSS Custom Properties
+Examples of what can be achieved are:
 
-Presets such as [color palettes](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-color-palettes), [font sizes](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-font-sizes), and [gradients](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-gradient-presets) become CSS Custom Properties, and will be enqueued by the system for themes to use in both the front-end and the editors. There's also a mechanism to create your own CSS Custom Properties.
+- Use a particular preset for a block (e.g.: table) but the common one for the rest of blocks.
+- Enable font size UI controls for all blocks that support it but the headings block.
+- etc.
 
 ### Some block styles are managed
 
-By providing the block style properties in a structured way, the Block Editor can "manage" the CSS that comes from different origins (user, theme, and core CSS), reducing the amount of CSS loaded in the page and preventing specificity wars due to the competing needs of the components involved (themes, blocks, plugins).
+By using the `experimental-theme.json` file to set style properties in a structured way, the Block Editor can "manage" the CSS that comes from different origins (user, theme, and core CSS). For example, if a theme and a user set the font size for paragraphs, we only enqueue the style coming from the user and not the theme's.
+
+Some of the advantages are:
+
+- Reduce the amount of CSS enqueued. 
+- Prevent specificity wars.
+
+### CSS Custom Properties
+
+There are some areas of styling that would benefit from having shared values that can change across a site instantly.
+
+To address this need, we've started to experiment with CSS Custom Properties, aka CSS Variables, in some places:
+
+- **Presets**: [color palettes](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-color-palettes), [font sizes](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-font-sizes), or [gradients](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-gradient-presets) declared by the theme are converted to CSS Custom Properties and enqueued both the front-end and the editors.
+
+{% codetabs %}
```

Comment:
I'm not sure this `codetabs` thing is going to work the way I expected it to work. This has been used to show ES5/ESNext code examples side by side. Can we use it for other things as well?

I thought I'd figure it out by pushing the changes to the handbook and see.

Reply:
Clever use, I think it will work. :+1:

Reply:
It didn't go very well :( I've created https://github.com/WordPress/gutenberg/pull/28655 to see if I can fix it. Comparing it to other `codetabs` uses, I don't see any missing markup.

Reply:
Using single words fixed it :muscle: